### PR TITLE
Allow multiple certs

### LINKED ISSUE #621: Allow multiple certificates / keys
When using skipper as entry point it would be nice to have multiple certificates. Go supports SNI on TLS enabled Servers. POC:

```go
package main

import (
	"os"
	"fmt"
	"net/http"
	"strings"
	"log"
	"crypto/tls"
)

func main() {
	certs := strings.Split(os.Args[1], ",")
	keys := strings.Split(os.Args[2], ",")

	cfg := &tls.Config{}
	for i, cert := range certs {
		kp, err := tls.LoadX509KeyPair(cert, keys[i])
		if err != nil {
			log.Fatalf("Failed to load key %s: %s", cert, err)
		}
		cfg.Certificates = append(cfg.Certificates, kp)
	}
	cfg.BuildNameToCertificate()

	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
		fmt.Fprintf(w, "Host: %s, TLS.ServerName: %s\n", r.Host, r.TLS.ServerName)
	})
	srv := &http.Server{
		Addr: ":8081",
		TLSConfig: cfg,
	}
	log.Fatalf("%s", srv.ListenAndServeTLS("",""))
}
```

Generate cert/keys with
```
go run $GOROOT/src/crypto/tls/generate_cert.go -host host1.local -ca
mv cert.pem host1.crt.pem
mv key.pem host1.key.pem
go run $GOROOT/src/crypto/tls/generate_cert.go -host host2.local -ca
```

edit /etc/hosts to set host1.local and host2.local as alias for 127.0.0.1  and run with `go run server.go host1.crt.pem,cert.pem host1.key.pem,key.pem` and use `curl -v https://host1.local:8081/ --cacert host1.crt.pem --http2` and `curl -v https://host2.local:8081/ --cacert cert.pem --http2`

### PR DESCRIPTION
**Pull request by vetinari**
allows multiple certs to be passed to skipper. The client selects the
cert with SNI.

fixes #621

### CODE REVIEW

File: skipper.go [hunk 1]
```diff
@@ -552,6 +555,25 @@ func listenAndServe(proxy http.Handler, o *Options) error {
...
+			tlsCfg := &tls.Config{}
+			crts := strings.Split(o.CertPathTLS, ",")
+			keys := strings.Split(o.KeyPathTLS, ",")
+			if len(crts) != len(keys) {
+				log.Fatalf("number of certs does not match number of keys")
+			}
+			for i, crt := range crts {
+				kp, err := tls.LoadX509KeyPair(crt, keys[i])
+				if err != nil {
+					log.Fatalf("Failed to load X509 keypair from %s/%s: %s", crt, keys[i], err)
```
Comment (szuecs) [inline skipper.go]: do %v for showing `err`

**Comment by szuecs** [comment]
SNI, sounds awesome

**Comment by aryszka** [comment]
:+1:

**Comment by vetinari** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**